src/components/KTabs/KTabs.vue

```diff
@@ -8,23 +8,24 @@
         v-for="(tab, i) in tabs"
         :id="`${tab.hash.replace('#','')}-tab`"
         :key="tab.hash"
-        :aria-controls="hasPanels ? `panel-${i}` : undefined"
-        :aria-selected="hasPanels ? (activeTab === tab.hash ? 'true' : 'false') : undefined"
         class="tab-item"
-        :class="{ active: activeTab === tab.hash }"
-        :role="hasPanels ? 'tab' : undefined"
-        tabindex="0"
-        @click="handleTabChange(tab.hash)"
-        @keydown.enter.prevent="handleTabChange(tab.hash)"
-        @keydown.space.prevent="handleTabChange(tab.hash)"
+        :class="{ active: activeTab === tab.hash, disabled: tab.disabled }"
       >
-        <div
-          :class="['tab-link', { 'has-panels': hasPanels }]"
+        <a
+          :aria-controls="hasPanels ? `panel-${i}` : undefined"
```

Comment:
question: I'm not sure we should be using the index as the identifier here; if there are multiple instances of tabs on the page, wouldn't this be invalid?